mlflow/pyfunc/__init__.py

```diff
@@ -2391,6 +2427,11 @@ def predict(model_input: List[str]) -> List[str]:
                 except Exception as e:
                     _logger.warning(f"Failed to infer model signature from input example. {e}")
 
+    # We loaded the code from model_code_path into python_model to infer the signature, but we
+    # don't want to save this python_model since the module won't be able to be imported.
+    if model_code_path:
+        python_model = None
```

Comment:
This means that code-based models can't support streaming. See https://github.com/mlflow/mlflow/pull/11951/files#r1603927952. Can we adopt the proposal in https://github.com/mlflow/mlflow/pull/11951/files#r1603927952 instead?

cc @sunishsheth2009